packages/demo/src/App.svelte

```diff
@@ -18,7 +18,10 @@
   <meta property="sesamy:price" content="1337" />
   <meta property="sesamy:currency" content="SEK" />
   <meta property="sesamy:publisher-content-id" content="some-id" />
-  <meta property="sesamy:item-src" content="some-id" />
+  <meta
```

Comment:
I think this works as a fallback, but we should probably recommend that it's set on the paywall or in some wrapper element. Otherwise we won't be able to have multiple paywalls on the same page.